### PR TITLE
Feature/dropdown text combo update

### PR DESCRIPTION
**Pull request by ajainarayanan**
Fixes syntactic error in bower.json. Don't know how I missed it.

### CODE REVIEW

File: modules/cask-angular-dropdown-text-combo/dropdown-text-combo.less [hunk 1]
```diff
@@ -1,3 +1,5 @@
+@import "../../bower_components/bootstrap/less/mixins/clearfix.less";
+@import "../../bower_components/bootstrap/less/variables.less";
```
Comment (gaarf) [inline modules/cask-angular-dropdown-text-combo/dropdown-text-combo.less]: I think we should point to the app's customized version of bootstrap "variables"...

```
@import "../../app/styles/variables.less";
```

Comment (gaarf) [inline modules/cask-angular-dropdown-text-combo/dropdown-text-combo.less]: as per our discussion, let's remove ref to `variables.less` and hardcode the gray color to be #eee.

Comment (ajainarayanan) [inline modules/cask-angular-dropdown-text-combo/dropdown-text-combo.less]: :+1: Done. Updated

**Comment by gaarf** [comment]
I think top-level bower.json in the ng-capsules repo is going to need to depend on bootstrap now, else the mixins.less will not be found.

**Comment by gaarf** [comment]
LGTM pending travis :+1:

**Pull request merged**